barustenberg/src/transcript.rs

```diff
@@ -692,7 +695,7 @@ impl<H: BarretenHasher, Fr: Field, G1Affine: AffineRepr> Transcript<H, Fr, G1Aff
     ) -> Fr {
         let idx = idx.unwrap_or(0);
         let buf = self.get_challenge(challenge_name, idx);
-        Fr::deserialize_uncompressed(buf.as_slice()).unwrap()
+        Fr::deserialize_uncompressed(buf.unwrap().as_slice()).unwrap()
```

Comment:
more unwraps?

Reply:
Left it because we don't have well defined error types yet. Which is what i wanted to start on next

Reply:
I like that idea